lecture2gether-vue/src/components/Lecture2GetherFooter.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <v-footer>
+        <v-spacer></v-spacer>
+        <img src="/img/icons/GitHub.svg" alt="GitHub logo" class="mr-2"/>
+        <a href="https://github.com/TheNerdful8/Lecture2Gether" class="footer-link">Lecture2Gether on Github</a>
+        <v-spacer></v-spacer>
+    </v-footer>
+</template>
+<script lang="ts">
+export default {
+    name: 'Lecture2GetherFooter',
+};
```

Comment:
A class component would be more consistent but since this component has no logic, it should be fine.

Reply:
I changed it anyway to be consistent, although it probably doesn't matter.